arca/backend/base.py

```diff
@@ -158,7 +151,9 @@ def run(self, repo: str, branch: str, task: Task, git_repo: Repo, repo_path: Pat
         try:
             logger.debug("Running with python %s", python_path)
 
-            process = subprocess.Popen([python_path, str(script_path.resolve())],
+            process = subprocess.Popen([python_path,
+                                        Path(__file__).parent.parent / "_runner.py",
```

Comment:
Does `"_runner.py"` reserve a constant such as `arca.RUNNER`?

Reply:
Yes, good idea! Actually, do you think it's a good idea to put the whole Path to the runner in the constant so subclasses could use a different runner by defining a path to a different file?

Reply:
why not